web/html-forms.md

```diff
@@ -25,7 +25,7 @@
 
 ### Lesson
 
-HTML Form is one of the very useful aspect which has vaste usage when some information about the user has to be stored. It can be said as a document where different kind of information such as name, email, blood group, contact number and other such information can be stored.
+HTML Form is one of the most popular ways to gather user information and send it to to an application. It is also the most correct. For example, it can be used to collect things like a name and email in a sign-up form.
```

Comment:
The line `It is also the most correct.` doesn't make any sense. So better to edit it to make it meaningful.